integrationTesting/tests/organize/views/detail/delete.spec.ts

```diff
@@ -1,34 +1,33 @@
+import test from '../../../../fixtures/next';
 import { expect, Page } from '@playwright/test';
-import test, { NextWorkerFixtures } from '../../../../fixtures/next';
 
 import AllMembers from '../../../../mockData/orgs/KPD/people/views/AllMembers';
 import AllMembersColumns from '../../../../mockData/orgs/KPD/people/views/AllMembers/columns';
 import AllMembersRows from '../../../../mockData/orgs/KPD/people/views/AllMembers/rows';
 import KPD from '../../../../mockData/orgs/KPD';
 
 const deleteView = async (page: Page) => {
-  await page.click('header [data-testid=ZUIEllipsisMenu-menuActivator]');
-  await page.click(`data-testid=ZUIEllipsisMenu-item-delete-view`);
-  await page.click('button:text("Confirm")');
-};
+  const menuButton = page.locator(
+    'header [data-testid=ZUIEllipsisMenu-menuActivator]'
+  );
+  const deleteButton = page.locator(
+    'data-testid=ZUIEllipsisMenu-item-delete-view'
+  );
+  const confirmButton = page.locator('button:text("Confirm")');
 
-const expectDeleteViewError = async (page: Page) => {
-  await page.locator('data-testid=Snackbar-error').waitFor();
-  const canSeeErrorSnackbar = await page
-    .locator('data-testid=Snackbar-error')
-    .isVisible();
-  expect(canSeeErrorSnackbar).toBeTruthy();
-};
+  await menuButton.waitFor({ state: 'visible' });
 
-const expectDeleteViewSuccess = (moxy: NextWorkerFixtures['moxy']) => {
-  const deleteViewRequest = moxy
-    .log()
-    .find(
-      (mock) =>
-        mock.method === 'DELETE' &&
-        mock.path === `/v1/orgs/1/people/views/${AllMembers.id}`
-    );
-  expect(deleteViewRequest).toBeTruthy();
+  await Promise.all([
+    menuButton.click(),
+    deleteButton.waitFor({ state: 'visible' }),
+  ]);
+
+  await Promise.all([
+    deleteButton.click(),
+    confirmButton.waitFor({ state: 'visible' }),
+  ]);
```

Comment:
Are these instances of `Promise.all()` necessary?

Reply:
Nope! BALEETED!